EAO/UI/Inspections/Uploadphoto/UploadPhotoController.swift

```diff
@@ -67,16 +67,17 @@ class UploadPhotoController: UIViewController, KeyboardDelegate{
     }
 
 	//MARK: -
-
 	@IBAction func backButtonTapped(_ sender: UIBarButtonItem) {
 		if didMakeChange{
-			present(controller: UIAlertController(title: "Would you like to save data?", message: nil, yes: {
-				self.save(nil)
-			}, cancel: {
-				self.pop()
-			}))
+            let alert = UIAlertController(title: "Would you like to save data?", message: nil, yes: {
+                self.save(nil)
+            }, cancel: {
+                self.popViewController()
+            })
+            
+			presentAlert(controller: alert)
 		} else{
-			pop()
+			popViewController()
```

Comment:
Unwind

Reply:
In this an all other cases below all the information is either sent back via callbacks or saved into realm before transitions. And since unwind segue is going to be called from code, I think that using it adds extra code and complexity without any benefits